examples/yt_camera/main.cpp

```diff
@@ -349,18 +369,33 @@ int main(int argc, char** argv)
             }
 
             // update background
+            if (enableBackground)
             {
                 Mat background;
                 pMOG2->getBackgroundImage(background);
+
+#if BACKGROUND_FROM_FILE
                 imwrite(BACKGROUND_TMP_FILE, background, compression_params); // save image
 
                 LOG(DEBUG) << "Posting background file " << BACKGROUND_TMP_FILE;
 
                 status = client.uploadBackground(accountId, instrumentId,
-                                                 prc::toTimestamp(prevMinuteStart), BACKGROUND_TMP_FILE);
+                                                 prc::toTimestamp(prevMinuteStart),
+                                                 prc::Payload(BACKGROUND_TMP_FILE));
+#else
+                std::vector<uchar> buf;
+                bool rv = imencode(".jpg", background, buf, compression_params);
```

Comment:
This is OpenCV jpeg encoding, right?
What it uses inside for it? Libjpeg?
We have to be max efficient.
We can have this in initial impl for demos, but eventually, we shall move to something super optimized.
Turbojpeg lib or HW encoder.